pkg/utils/release/controller.go

```diff
@@ -424,3 +450,131 @@ func (s *SuiteController) CreateReleasePipelineRoleBindingForServiceAccount(name
 	}
 	return roleBinding, nil
 }
+
+// WaitForReleasePipelineToBeFinished Waits for the release pipelineRun associated to the release and component to finish within a given retries.
```

Comment:
Well, there are checks about if the release string length is greater than 0, so this isn't really accurate